spec/index.bs

```diff
@@ -1801,8 +1802,12 @@ dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
     maximum levels of detail, respectively, used internally when sampling a texture.
 - If {{GPUSamplerDescriptor/compare}} is provided, the sampler will be a comparison sampler with the specified
     {{GPUCompareFunction}}.
+- {{GPUSamplerDescriptor/maxAnisotropy}} specifies the maximum anisotropy value clamp used by the sampler.
+
+Note: most implementations support {{GPUSamplerDescriptor/maxAnisotropy}} values in range between 1 and 16, inclusive.
```

Comment:
I think it should work if it's indented enough (this list is only indented by 1 but it's probably treated as 4?, so the next line might need 6 or 8)

Or you can fix the list so it's indented by 4 and the continuations are indented by 8.
```suggestion
        Note: most implementations support {{GPUSamplerDescriptor/maxAnisotropy}} values in range between 1 and 16, inclusive.
```

Reply:
Actually this list is not indented so it can probably go down to 0 and have the Note be at 4.